salt/profiles/config/srv-profiles-app.cfg

```diff
@@ -8,6 +8,8 @@ authorize_uri = {{ pillar.profiles.orcid.authorize_uri }}
 token_uri = {{ pillar.profiles.orcid.token_uri }}
 client_id = {{ pillar.profiles.orcid.client_id }}
 client_secret = {{ pillar.profiles.orcid.client_secret }}
+public_access_token = {{ pillar.profiles.orcid.public_access_token }}
```

Comment:
`read_public_access_token` (to match the scope)?

Reply:
Done.